README.md

```diff
@@ -7,9 +7,19 @@ Smart contracts for C4Coin's forthcoming crowdsale
 
 ## Overview
 
-It is proposed that C4Coin will launch a [Regulation A+](https://en.wikipedia.org/wiki/Regulation_A#Regulation_A+) crowdsale, granting buyers of its tokens stock in the company.
+It is proposed that C4Coin will launch a [Regulation A+](https://en.wikipedia.org/wiki/Regulation_A#Regulation_A+) compatible token, where each issued token corresponds to a unit of stock in the company.
```

Comment:
Can we clarify here if we are going for common stock (with voting rights) or preferred stock?

Reply:
common stock is a good clarification